packages/cli/src/modules/breaking-changes/rules/v2/queue-worker-max-stalled-count.rule.ts

```diff
@@ -0,0 +1,58 @@
+import { GlobalConfig } from '@n8n/config';
+import { Service } from '@n8n/di';
+
+import type {
+	BreakingChangeRuleMetadata,
+	IBreakingChangeInstanceRule,
+	InstanceDetectionReport,
+} from '../../types';
+import { BreakingChangeCategory } from '../../types';
+
+@Service()
+export class QueueWorkerMaxStalledCountRule implements IBreakingChangeInstanceRule {
+	constructor(private readonly globalConfig: GlobalConfig) {}
+
+	id: string = 'queue-worker-max-stalled-count-v2';
+
+	getMetadata(): BreakingChangeRuleMetadata {
+		return {
+			version: 'v2',
+			title: 'Remove QUEUE_WORKER_MAX_STALLED_COUNT',
+			description:
+				'The QUEUE_WORKER_MAX_STALLED_COUNT environment variable has been removed and will be ignored',
+			category: BreakingChangeCategory.environment,
+			severity: 'low',
+			documentationUrl:
+				'https://docs.n8n.io/2-0-breaking-changes/#remove-queue_worker_max_stalled_count',
+		};
+	}
+
+	async detect(): Promise<InstanceDetectionReport> {
+		const result: InstanceDetectionReport = {
+			isAffected: false,
+			instanceIssues: [],
+			recommendations: [],
+		};
+
+		// Check if QUEUE_WORKER_MAX_STALLED_COUNT is set to a value other than the default (1)
+		if (this.globalConfig.queue.bull.settings.maxStalledCount === 1) {
```

Comment:
I think here as well we can check if `QUEUE_WORKER_MAX_STALLED_COUNT` has been configured at all and only report then